jest.config.js

```diff
@@ -7,13 +7,16 @@ module.exports = {
   },
   moduleNameMapper: {
     '\\.(css|less|scss|sass)$': 'identity-obj-proxy',
-    '^\\./desktop/core/src/desktop/js/(.*)$': '<rootDir>/desktop/core/src/desktop/js/$1'
+    '^\\./desktop/core/src/desktop/js/(.*)$': '<rootDir>/desktop/core/src/desktop/js/$1',
+    '^@vue/test-utils': '<rootDir>/node_modules/@vue/test-utils/dist/vue-test-utils.cjs.js'
```

Comment:
What was this for?

Reply:
Either a bug or a breaking change in the Vue test-utils caused it not to find Vue. Solution from https://stackoverflow.com/questions/72428323/jest-referenceerror-vue-is-not-defined